app/views/spotlight/sir_trevor/blocks/_featured_pages_block.html.erb

```diff
@@ -3,7 +3,7 @@
     <% featured_pages_block.pages.each_with_index do |page, index| %>
       <div class="box category-<%= (index + 1) %>">
         <%= link_to [page.exhibit, page] do %>
-          <div class="browse-category" style='background-image: linear-gradient(rgba(0, 0, 0, 0.0), rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.5)) <%= ", url(\"#{page.thumbnail.image.thumb}\")" if page.thumbnail %>'>
+          <div class="browse-category" style='background-image: linear-gradient(rgba(0, 0, 0, 0.0), rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.5)) <%= ", url(\"#{riiif.image_path(page.thumbnail, size: Spotlight::Engine.config.featured_image_thumb_size.join(','))}\")" if page.thumbnail %>'>
```

Comment:
This should probably be using the thumbnail's `iiif_url` instead of this (which won't account for crop coordinates)